assessments/unit-01-booleans-and-strings/code.js

```diff
@@ -9,3 +9,31 @@ const rand = (n) => Math.floor(Math.random() * n);
 // syntactically correct function skeleton, more credit for a skeleton with a
 // reasonable argument list, yet more credit for a working function, and full
 // credit for a clear and simple working function.
+
+const fireAlarm = (pulled,smoke,planned) => {
+  return !pulled || !smoke || !planned
+}
+
+const canBePresident = (age,citizen,years) => {
+  if (age >= 35, citizen = true, years >= 14) {
+    return true
+  } else {
+    return false
+  }
+}
+
+const firstAndLast = (string) => {
+  return string[0]+string[string.length - 1]
+}
+
+const swapFrontAndBack = (string) => {
+  return string.substring(string.length/2, string.length) + string.substring(0,string.length/2)
+}
+
+const simplePigLatin = (string,number) => {
+  return string.substring(number,string.length)+string.substring(0,number-1)+'ay'
+}
+
+const randomCharacter = (string) => {
+  return string[rand]
```

Comment:
`rand` is a function so you need to call it meaning there need to be `()`s and, in this case, a single argument.

```suggestion
  return string[rand(string.length)]
```